.github/workflows/tests.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       - name: Run hex audit
         run: mix hex.audit
       - name: Run mix_audit
-        run: mix deps.audit --ignore-advisory-ids GHSA-vq52-99r9-h5pw
+        run: mix deps.audit --ignore-advisory-ids GHSA-vq52-99r9-h5pw,GHSA-9fm9-hp7p-53mf
```

Comment:
The dependency doesn't seem to be used so no risk here. To be removed here https://github.com/supabase/realtime/pull/1390